src/code/z_scene.c

```diff
@@ -492,10 +466,32 @@ void func_800993C0(GlobalContext* globalCtx, SceneCmd* cmd) {
 }
 
 void (*gSceneCmdHandlers[SCENE_CMD_ID_MAX])(GlobalContext*, SceneCmd*) = {
-    func_80098508, func_800985DC, func_80098630, func_80098674, func_800987A4, func_80099090, func_800987F8,
-    func_8009883C, func_80098904, func_80099134, func_80098958, func_8009899C, func_80098B74, func_80098C24,
-    func_80098C68, func_80098CC8, func_80098D80, func_80098D1C, func_80098D5C, func_800990F0, NULL,
-    func_80099140, func_8009918C, func_8009934C, func_800991A0, func_800993C0,
+    Scene_CommandSpawnList,           // SCENE_CMD_ID_SPAWN_LIST
+    Scene_CommandActorList,           // SCENE_CMD_ID_ACTOR_LIST
```

Comment:
for my planned cleanup of the cutscene system, I was planning on doing `CutsceneCmd_Name`
if this is something people are on board with, we can do this for all functions that handle processing commands from a script

Reply:
`SceneCmd_Name` sounds good to me, waiting for other opinions
(it would make it one character different from the struct names I'd want to use `SceneCmdName` (the `_` only) but we probably don't care)

Reply:
I'd definitely prefer `[SystemName]Cmd_[X]` over `[SystemName]_Command[X]`, imo it reads better if there's space after `Command` and the command name.
Probably unpopular but it might even be nice to do `[SystemName]_Cmd_[X]`, the prefix stays the same as other system functions but benefits from the extra gap before the command name

Reply:
I think the main system name being slightly varied helps get there point across better that it's a cutscene handler not meant to be called externally. It's also just more aesthetically pleasing to me. But yeah I would want System_Command_Name over System_CommandName if it came down to those two

Reply:
I wouldn't say that adding to the system prefix generically gives the impression that it's not meant to be called externally (of course there's a surefire way to do that, that we don't want to use). I'd agree that one of the options that separates the `Cmd` from the subsequent description is better, and `SceneCmd_Name` is more in line with what we usually do.

Reply:
I mean, making the command functions have its own system name such as `SceneCmd` helps it to separate it from the the normal scene functions, which helps get the point across that they arent meant to be called on theor own. So I would say that it does help.

But yeah, I do agree that it is more in line with what we usually do anyway